features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/data/api.jsx

```diff
@@ -319,7 +319,7 @@ export default class API extends Resource {
      */
     getAllComments(apiId, callback = null) {
         const promiseGet = this.client.then((client) => {
-            return client.apis['Comment (Collection)'].get_apis__apiId__comments({ apiId }, this._requestMetaData());
+            return client.apis['Comments'].get_apis__apiId__comments({ apiId }, this._requestMetaData());
```

Comment:
We could use `Comments` name directly as below
```suggestion
            return client.apis.Comments.get_apis__apiId__comments({ apiId }, this._requestMetaData());
```

Reply:
If i change like this i was getting the same error actually..

Reply:
mm , since this is a valid JS variable name, it should be able to access without quoting. Nonetheless, it's not a big issue 😃